backend/fms_core/template_importer/row_handlers/normalization/normalization.py

```diff
@@ -46,8 +46,10 @@ def process_row_inner(self, source_sample, destination_sample, process_measureme
             # Case when nM is given
             elif destination_sample['concentration_nm']:
                 if source_sample_obj.is_library:
+                    # TODO: Adapt so we use avg. library size of a pool
                     library = source_sample_obj.derived_sample_not_pool.library
                     if library.library_size:
+                        # TODO: improve conversion service to include pool of libraries
```

Comment:
Use same service as lib_qc (convert_sample_concentration_from_nm_to_ngbyul(sample, concentration). We have to remember to have an comparable service for the reverse operation for the frontend.